### PR TITLE
Redshift connector

### LINKED ISSUE #456: Redshift connector docs

### PR DESCRIPTION
**Pull request by eeshansim**
fixes #456

### CODE REVIEW

File: connectors/redshift/updated-row-trigger.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+  </tbody>
+</table>
+
+## Input fields
+
+### Table
+Select the table/view to process rows from. This can be done either by selecting a table from the pick list, or toggling the input field to text mode and typing the full table name.
+
+### Unique key
+Values from this selected column is used to deduplicate rows in the selected table.
```
Comment (jandonyada) [inline connectors/redshift/updated-row-trigger.md]: Values from this selected column are

File: connectors/redshift/updated-row-trigger.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+As such, the values in the selected column should not be repeated in your table. Typically, this column is the primary key of the table (e.g. `ID`). It should be incremental and sortable. This column can also be indexed for better performance.
+
+### Sort column
+Sort column is a column that is updated whenever a row in the table is updated. Typically, this is a timestamp column.
+
+When a row is updated, the Unique key value remains the same. However, it should have it's timestamp updated to reflect the last updated time. Following this logic, Workato keeps track of values in this column together with values in the selected [**Unique key**](#unique-key) column. When a change in the **Sort column** value is observed, an updated row event will be recorded and processed by the trigger.
```
Comment (jandonyada) [inline connectors/redshift/updated-row-trigger.md]: Should we standardize whether to use **Unique key**, Unique key (both used in this line), or unique key (used in several other parts of the docs), or is it intended to separate unique keys, Unique key values, and **Unique key** columns?

File: connectors/redshift/insert.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+![A list datapill from the datatree](/assets/images/redshift/list_datapill_in_output_tree.png)
+*A list datapill from the datatree*
+
+If you do not map a list datapill to this field, this action will insert only 1 row and will behave like the **Insert row** action.
+
+### Columns
+Finally, you will need to map the required fields from the output datatree here to insert rows with data from preceding trigger or actions. Take note that datapills mapped to each column here should be from the source list datapill you used earlier. Datapills that are mapped outside the source list datapill will not be iterated.
+
+Refer to the [List management](/features/list-management.html) guide for more information about working with batches.
```
[the remark this answers is not in the document]
Comment (eeshansim) [reply connectors/redshift/insert.md]: Will change for other pages too.

File: connectors/redshift/updated-row-trigger.md (hunk 2 above)
Comment (eeshansim) [reply connectors/redshift/updated-row-trigger.md]: Good point. Bolded **Unique key** refers to the input field, and normally styled Unique key refers to the col-row value that lives in the users database.

Does it create more confusion? I'm ok with bolding all instances of Unique Key.

File: connectors/redshift/select.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+### Order by
+Rows returned from this action can be ordered based on the **Order by** input field. This field is used to change the default ordering of rows from your Redshift database.
+
+You can also define the direction of order for each column you wish to order by. The following order by statement will order rows by `priority` in ascending order followed by `created_date` in descending order (latest first).
+
+```sql
+priority asc, created_date desc
+```
+
```
[the remark this answers is not in the document]
Comment (eeshansim) [reply connectors/redshift/select.md]: added

**Pull request merged**